### ISSUE TITLE
[Feature] Hide tooltip option

### ISSUE THREAD

**Issue by L9Tigrotto** [yanggx98/Immersive-Tooltip#11]
I'd like to get rid of kaleido tooltip, any option i'm missing?

![image](https://github.com/user-attachments/assets/72d69431-25d4-48eb-8f48-946fbe857929)

**Comment by yanggx98** [comment]
This seems to be another tooltip mod. If you want to use it,you can delete kaleido and Immersive-Tooltip.

**Comment by L9Tigrotto** [comment]
Yes it's [Crital](https://modrinth.com/mod/crital) for the custom weapons tool tip and [Durability Tooltip](https://modrinth.com/mod/durability-tooltip) for colored durability. I'd like to use the Crital's tool tip and Durability Tooltip's durability on weapons and armours (without the Kaleido durability), but keep the Immersive-Tooltip's tool tip for all other items.

**Comment by yanggx98** [comment]
> Yes it's [Crital](https://modrinth.com/mod/crital) for the custom weapons tool tip and [Durability Tooltip](https://modrinth.com/mod/durability-tooltip) for colored durability. I'd like to use the Crital's tool tip and Durability Tooltip's durability on weapons and armours (without the Kaleido durability), but keep the Immersive-Tooltip's tool tip for all other items.

ok,the next version will be added. thank you for liking this mod.
